### DISCUSSION TITLE
Incorporation of bond endpoints

### DISCUSSION THREAD

**Discussion by MoSafi2** [joqueka/bf4py#15] [Ideas]
First, I want to thank immensly for making this available. 
I am looking to use the API for getting corporate bond information and bond searchs. 
I found only one point for bond information which is /master_data_bond 
and other endpoints as /instrument_information and /price_history also return information. 
Is there more endpoints for bond search? by issuer for example? 
Also, It would be great to add a bond module to this API.

**Comment by joqueka** [comment]
Hi
from my notes I made a longer time ago I know there's and endpoint for bond search but I didn't inspected it in detail so I don't know which search parameters it uses. /master_data_bond is new to me. I will have look at both endpoints and will implement them when I find some time for this. If you don't want to wait you can try it on yourself of course and make a pull request.

**Comment by joqueka** [comment]
Hi
I just added some methods for searching bonds, they're in the submodule bf4py.bonds. Besides the search you can use many regular methods from bf4py.equities to get detailed information about a bond.
